docs/docs/get_started.md

```diff
@@ -6,8 +6,8 @@ description: Secretless Broker Documentation
 permalink: docs/get_started.html
 ---
 
-To get started trying out the Secretless Broker, you can run through our [quick start demos](/docs/get_started/quick_start.html)
-that allow you to try it out with PostgreSQL, SSH, or an HTTP web service.
+To get started trying out the Secretless Broker, you can run through our [quick start](/docs/get_started/quick_start.html)
+that allows you to try it out with a simple example using PostgreSQL, SSH, or an HTTP web service.
```

Comment:
this sounds awkward - maybe just "that allows you to try some simple examples using"

Reply:
i like it. Will update